sections/semantics-forms.include

```diff
@@ -10489,29 +10489,53 @@ You cannot submit this form when the field is incorrect.</samp></pre>
   <hr />
 
   The <dfn element-attr for="form"><code>enctype</code></dfn> and
-  <dfn element-attr for="submitbuttonelements,input,button"><code>formenctype</code></dfn> content attributes are <a>enumerated attributes</a> with the following keywords and
-  states:
+  <dfn element-attr for="submitbuttonelements,input,button"><code>formenctype</code></dfn>
+  content attributes are <a>enumerated attributes</a> that specify how form data should be encoded
+  upon a form submission. These attributes have the following keywords and states:
 
-  <ul>
-    <li>The "<dfn attr-value for="form/enctype"><code>application/x-www-form-urlencoded</code></dfn>" keyword and corresponding state.</li>
-    <li>The "<dfn attr-value for="form/enctype"><code>multipart/form-data</code></dfn>" keyword and corresponding state.</li>
-    <li>The "<dfn attr-value for="form/enctype"><code>text/plain</code></dfn>" keyword and corresponding state.</li>
-  </ul>
+  <dl>
+    <dt>The
+      "<dfn attr-value for="form/enctype"><code>application/x-www-form-urlencoded</code></dfn>"
+      keyword and corresponding state.
+    </dt>
+    <dd>
+      The default value if the attribute is not specified.
+    </dd>
+
+    <dt>
+      The "<dfn attr-value for="form/enctype"><code>multipart/form-data</code></dfn>"
+      keyword and corresponding state.
+    </dt>
+    <dd>
+      Allows for entire files to be submitted along with with form data.
+    </dd>
+
+    <dt>
+      The "<dfn attr-value for="form/enctype"><code>text/plain</code></dfn>"
+      keyword and corresponding state.
+    </dt>
+    <dd>
+      Submits as <a>plain text form data</a>, which is meant to be human readable.
```

Comment:
This anchor doesn't work... if it's defined in `<h5 id="plain-text-form-data">Plain text form data</h5>`, perhaps we should use `<a href="#plain-text-form-data">` instead.

Reply:
or make the term there a definition: `<h5><dfn>plain...`